src/devices/sound/pokey.cpp

```diff
@@ -594,37 +641,28 @@ void pokey_device::step_one_clock(void)
 			step_keyboard();
 	}
 
-	/* do CHAN2 before CHAN1 because CHAN1 may set borrow! */
-	if (m_channel[CHAN2].check_borrow())
+	if (m_channel[CHAN3].check_borrow())
 	{
-		bool const isJoined(m_AUDCTL & CH12_JOINED);
+		int isJoined = (m_AUDCTL & CH34_JOINED);
 		if (isJoined)
```

Comment:
If we're going to bother changing this can we just nix these temporary `isJoined` variables that aren't reused? That's consistent with a bunch of other places in the code and the mask names tell us all we need to know anyway. So here we could just do `if (m_AUDCTL & CH34_JOINED)`.